src/modules/cache-client/cache-client.service.ts

```diff
@@ -72,43 +73,26 @@ export class CacheClient implements ICacheClient {
    * After authentication runs previously failed requests
    */
   async authenticate() {
-    let tokens: AuthTokens | undefined = undefined;
-
-    const setTokens = () => {
-      if (!tokens) return;
-      if (!this.isBrowser) {
-        this._httpClient.defaults.headers.common[
-          'Authorization'
-        ] = `Bearer ${tokens.token}`;
-      }
-      this.refresh_token = tokens.refreshToken;
-    };
-
     // First try to refresh access token
     try {
-      const refreshedTokens = await this.refreshToken();
-      tokens = refreshedTokens;
-      setTokens();
-
-      if (!tokens || !(await this.isAuthenticated())) {
-        tokens = undefined;
-      }
+      await this.refreshToken();
```

Comment:
why not log error details here? We very often suffer because of no details when debugging

Reply:
Ok, could be usefull